### PR TITLE
Move NumericalTypesTest to community.

### PR DESCRIPTION
**Pull request by kkufova**
This PR was created to move the [NumericalTypesTest Selenium test](https://github.com/jboss-integration/bxms-qe-tests/blob/master/test-kie-wb/src/test/java/org/jboss/qa/brms/authoring/datamodeler/feature/NumericalTypesTest.java) from our internal test suite to community.

@jomarko, @manstis, could you please review?

### CODE REVIEW

**Comment by kkufova** [comment]
@manstis, thanks for noticing, fixed!

**Comment by kkufova** [comment]
jenkins please retest this

File: drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/java/org/kiegroup/NumericalTypes.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    private Float floatField;
+    private Integer integerField;
+    private Long longField;
+    private Short shortField;
+    private String result;
+
+    public NumericalTypes() {
+    }
+
+    public NumericalTypes(
```
Comment (manstis) [inline drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/java/org/kiegroup/NumericalTypes.java]: Not sure if our check-styles plugin will moan about the constructor parameters not starting on the same line as the opening parenthesis but let's wait and see (AFAIK our code styles have the first parameter on the same line as the opening parenthesis).

Comment (kkufova) [reply drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/src/test/resources/org/drools/workbench/screens/drltext/backend/server/drl/src/main/java/org/kiegroup/NumericalTypes.java]: @manstis, I fixed it, I don't want your preferences to be violated either. :-)

**Comment by kkufova** [comment]
@manstis, the build is green, could you please merge?

**Pull request merged**